modules/virt-switching-displays.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-configuring-mediated-devices.adoc
+
+:_content-type: PROCEDURE
+[id="virt-switching-displays_{context}"]
+= Switching between VM displays
+
+If your virtual machine (VM) has multiple displays, you can view each one by switching displays in the web console.
+
+.Prerequisites
+
+* The mediated device is configured in the `HyperConverged` custom resource and assigned to a VM.
+
+.Procedure
+
+. In the OpenShift Virtualization console, click *Virtualization* in the side menu.
+. Click the *Virtual Machines* tab.
+. Select a Windows virtual machine to open the *Overview* screen.
+. Click the *Console* tab.
+. Select the appropriate display from the *Send Key* drop down menu or by using the tabs below the drop down menu. Use send key combination `Alt+Ctl+1` for the first display and `Alt+Ctl+2` for the second display.
```

Comment:
According to the ISG, refer to the drop-down list by its label followed by the word "list".

Also, do not use location specific words such as "below" for accessibility requirements. Which tabs are you referring to below the drop-down menu? I didn't see anything that meets that description in the test environment.

I would suggest rewriting this step as something like this:

````
. Choose the appropriate key combination from the *Send Key* list:
.. To access the default VM display, select `Ctl + Alt+ 1`.
.. To access the vGPU display, select `Ctl + Alt + 2`.
````

Reply:
There's a new bug re: the tabs, so I'm going to remove that language for now as it doesn't look like it will be repaired in time for 4.11. We can then add the functionality once it's working as expected 👍 .